python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -629,14 +634,18 @@ def __init__(self, initial_stars_schema=None, **kwargs):
                            "calib_astrometry_used",
                            # TODO DM-39203: these can be removed once apcorr is gone.
                            "apcorr_slot_CalibFlux_used", "apcorr_base_GaussianFlux_used",
-                           "apcorr_base_PsfFlux_used")
+                           "apcorr_base_PsfFlux_used",)
         for field in self.psf_fields:
             item = self.psf_schema.find(field)
             initial_stars_schema.addField(item.getField())
         id_type = self.psf_schema["id"].asField().getTypeString()
+        psfmaxvaluetype = self.psf_schema['psf_max_value'].asField().getTypeString()
```

Comment:
```suggestion
        psf_max_value_type = self.psf_schema['psf_max_value'].asField().getTypeString()
```

Reply:
Also, why is this the only place to infer the value type when in other cases they are always set to `float32`?